### PR TITLE
#169974682 Initialize app

### PR DESCRIPTION
**Pull request by chuxmykel**
#### What does this PR do?
- This PR sets up a basic Express app for the platform to build on.

#### Description of Task to be completed?
- create a basic express app
- integrate dotenv to handle environment variables
- create GitHub PR template
- document app startup procedure in README.md

#### How should this be manually tested?
- Follow the instructions in the `README.md` file to set up the app and then make a GET request to the `/api/v1/` endpoint. 

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#169974682](https://www.pivotaltracker.com/story/show/169974682)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# schools_service_v1
\ No newline at end of file
+# schools_service_v1
```
Comment (EdustripeDev) [inline README.md:1]: Great work starting documentation from the jump 👍🏽

File: build/app.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+"use strict";
```
Comment (EdustripeDev) [inline build/app.js]: The build folder still made it here despite being in the gitignore file, is this deliberate?

**Review by tolulope-od — DISMISSED** [review]
asides the comments pointed out LGTM

File: build/app.js (hunk 2 above)
Comment (chuxmykel) [reply build/app.js]: Oops! I'll take it out now.

Comment (tolulope-od) [reply build/app.js]: Great!

**Review by EdustripeDev — APPROVED** [review]

**Review by tolulope-od — APPROVED** [review]

**Review by tolulope-od — APPROVED** [review]

**Pull request merged**